{"name": "test_login", "status": "failed", "statusDetails": {"message": "AssertionError: 'Epic sadface: Sorry, this user has been locked out...' != 'Epic sadface: Sorry, this user has been locked out.'\n- Epic sadface: Sorry, this user has been locked out...\n?                                                    --\n+ Epic sadface: Sorry, this user has been locked out.\n : The actual result does not the same with the expected result.", "trace": "self = <TestCases.test_login_03.Login03 testMethod=test_login>\n\n    def test_login(self):\n        account = Account('locked_out_user', 'secret_sauce')\n        login_page = LoginPage(self.browser)\n    \n        # 3 ways to reduce the code\n        login_page.login(account)\n>       self.assertEqual('Epic sadface: Sorry, this user has been locked out...', login_page.get_error_message(),\n                         'The actual result does not the same with the expected result.')\nE       AssertionError: 'Epic sadface: Sorry, this user has been locked out...' != 'Epic sadface: Sorry, this user has been locked out.'\nE       - Epic sadface: Sorry, this user has been locked out...\nE       ?                                                    --\nE       + Epic sadface: Sorry, this user has been locked out.\nE        : The actual result does not the same with the expected result.\n\nTestCases/test_login_03.py:26: AssertionError"}, "start": 1622031823303, "stop": 1622031833334, "uuid": "7dddf64b-60dc-435f-bea9-a38e1ccd09eb", "historyId": "c9f2eea7007d29364cd923079c7f9801", "testCaseId": "16e2b98178e8b4c23691068f67b40028", "fullName": "TestCases.test_login_03.Login03#test_login", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login_03"}, {"name": "subSuite", "value": "Login03"}, {"name": "host", "value": "Dum-GeoTechs-Mac-mini.local"}, {"name": "thread", "value": "23566-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login_03"}]}